app/src/main/java/eu/kanade/presentation/library/components/LibraryToolbar.kt

```diff
@@ -107,6 +110,7 @@ private fun LibraryRegularToolbar(
         searchQuery = searchQuery,
         onChangeSearchQuery = onSearchQueryChange,
         actions = {
+            val syncPreferences: SyncPreferences = Injekt.get() // SY
```

Comment:
The `syncPreferences.isSyncEnabled()` needs to be a state thats stored inside the LibraryScreenModel and passed through a parameter. We can't have it inside the composable